casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
```

Comment:
So I know I reluctantly agreed to use minutes for timing on earlier PR but at that point I didn't know we were going to use them for timing message replies. Replies can happen within minutes, so it seems needlessly inaccurate to round everything up to the nearest minute. Changing this now to something like `DailyTiming` that uses seconds will be a lot easier than changing it later - so can we do that?